src/test/java/org/mastodon/StartImageJ.java

```diff
@@ -38,7 +38,6 @@
  */
 public class StartImageJ
 {
-
```

Comment:
This change feels unnecessary. Maybe remove the commit, if you already know how to do this.

Reply:
I removed the commit.